products/iambeta/api.yaml

```
@@ -94,11 +94,199 @@ objects:
+          function that maps an identity provider credential to the normalized attribute specified
+          by the corresponding map key.
+
+          You can use the `assertion` keyword in the expression to access a JSON representation of
+          the authentication credential issued by the provider.
+
+          The maximum length of an attribute mapping expression is 2048 characters. When evaluated,
+          the total size of all mapped attributes must not exceed 8KB.
+
+          For AWS providers, the following rules apply:
+            - If no attribute mapping is defined, the following default mapping applies:
+              ```
+              {
+                "google.subject":"assertion.arn",
+                "attribute.aws_role":
+                  "assertion.arn.contains('assumed-role')"
+                  " ? assertion.arn.extract('{account_arn}assumed-role/')"
+                  "   + 'assumed-role/'"
+                  "   + assertion.arn.extract('assumed-role/{role_name}/')"
+                  " : assertion.arn",
+              }
+              ```
+            - If any custom attribute mappings are defined, they must include a mapping to the
+              `google.subject` attribute.
+
+          For OIDC providers, the following rules apply:
+            - Custom attribute mappings must be defined, and must include a mapping to the
+              `google.subject` attribute. For example, the following maps the `sub` claim of the
+              incoming credential to the `subject` attribute on a Google token.
+              ```
+              {"google.subject": "assertion.sub"}
+              ```
+      - !ruby/object:Api::Type::String
+        name: 'attributeCondition'
+        description: |
+          [A Common Expression Language](https://opensource.google/projects/cel) expression, in
+          plain text, to restrict what otherwise valid authentication credentials issued by the
+          provider should not be accepted.
+
+          The expression must output a boolean representing whether to allow the federation.
+
+          The following keywords may be referenced in the expressions:
+            * `assertion`: JSON representing the authentication credential issued by the provider.
+            * `google`: The Google attributes mapped from the assertion in the `attribute_mappings`.
+            * `attribute`: The custom attributes mapped from the assertion in the `attribute_mappings`.
+
+          The maximum length of the attribute condition expression is 4096 characters. If
+          unspecified, all valid authentication credential are accepted.
+
+          The following example shows how to only allow credentials with a mapped `google.groups`
+          value of `admins`:
+          ```
+          "'admins' in google.groups"
+          ```
+      - !ruby/object:Api::Type::NestedObject
+        name: aws
+        description: An Amazon Web Services identity provider.
```

```suggestion
        description: An Amazon Web Services identity provider. Not compatible with the property oidc.
```
Should be something describing the relationship here